extensions/amp-sidebar/1.0/sidebar.js

```diff
@@ -123,13 +132,28 @@ function SidebarWithRef(
     [open, close, toggle]
   );
 
+  useLayoutEffect(() => {
+    if (side) {
+      return;
+    }
+    const sidebarElement = sidebarRef.current;
+    if (!sidebarElement) {
+      return;
+    }
+    setSide(isRTL(sidebarElement.ownerDocument) ? Side.RIGHT : Side.LEFT);
+  }, [side, mounted]);
```

Comment:
Only reaches here if `sideProp` is undefined so we can directly use `document.dir` to set `side`